layers/+lang/java/packages.el

```diff
@@ -198,7 +198,6 @@
           "br"     'ensime-sbt-do-run
 
           "ct"     'ensime-typecheck-current-buffer
-          "cT"     'ensime-typecheck-all
```

Comment:
Are these also removed from the stable release? The recommendation is to pin ensime to stable melpa and the functionality might still be there... I'm not sure how to handle this, or even if we should handle it.

Reply:
Yep, there are all removed from the package on stable melpa.